app/models/company.rb

```diff
@@ -1,6 +1,7 @@
 class Company < ApplicationRecord
- validates :name, :address, :phone, :category_id, :town_id, presence: true
+ validates :name, :address, :phone, :category_id, :town_id, :user_id, presence: true
 
+  belongs_to :user
```

Comment:
I think you also need to add the relation in the [User](https://github.com/BC-MAY-21-ROR/proyecto-proy-t02/blob/master/app/models/user.rb#L1) controller...

Reply:
we'll add it